skynet/dgpu/tui.py

```diff
@@ -0,0 +1,211 @@
+import json
+import logging
+import warnings
+
+import trio
```

Comment:
unused import says `ruff`

Reply:
Removed in `structify` branch (coming soon)